packages/cyberstorm/src/components/Header/HeaderUserNav.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+import { faUpload } from "@fortawesome/pro-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useDapper } from "@thunderstore/dapper";
+import { usePromise } from "@thunderstore/use-promise";
+
+import * as Button from "../Button/";
+import { UserDropDown } from "./UserDropDown";
+
+export const HeaderUserNav = () => {
```

Comment:
I don't think these should necessarily be in the same component even though they benefit from sharing (a cached) Dapper request. Conceptually they have little to do with each other. But I guess changing this later won't be any more work than changing it now so might as well do that later if we see it necessary.